test/string_util_gtest.cc

```diff
@@ -167,12 +167,13 @@ using AppendHumanReadableFixture =
 INSTANTIATE_TEST_SUITE_P(
```

Comment:
Looks like the only test coverage we get is via a change to the `AppendHumanReadable()`,
which isn't used anywhere else other than this test...
I would have expected to see some test for the console reporter to fail.

Reply:
i fixed the console reporter when i updated HumanReadableNumber.

i didn't spot that this method isn't used anywhere... i guess we should just delete it then if it's dead code!

Reply:
So, should I remove this function and the test case related to this? And validate if it can be safely removed.

Reply:
i believe so.. sorry for all the back and forth!

Reply:
No, Dominic, I actually learned so much. :)